protocols/0556be-covid-ntc/README.md

```diff
@@ -0,0 +1,44 @@
+# COVID NTC Protocol
+
+### Author
+[Opentrons](https://opentrons.com/)
+
+## Categories
+* Sample Prep
+	* Cherrypicking
```

Comment:
I would change the category here from `Sample Prep` --> `Cherrypicking` to `Sample Prep` --> `Plate Filling`